lib/github.rb

```diff
@@ -59,7 +59,12 @@ def commits_count
   # @return [Fixnum] the number of pull request comments
   def pull_request_comments_count
     events
-      .select { |event| event.type == 'PullRequestReviewCommentEvent' }
+      .select { |event|
```

Comment:
Fair enough, I also like it more, I somehow thought that https://github.com/bbatsov/ruby-style-guide#single-line-blocks suggested otherwise... :shipit: